omego/artifacts.py

```diff
@@ -472,6 +475,56 @@ def read_downloads(dlurl):
         return dl_icever
 
 
+class GithubArtifacts(ArtifactsList):
```

Comment:
Could be renamed as `GitHubArtifacts` but not a blocker

Reply:
I was undecided on following the `GitHub` branding vs `CapitalisedsubtypeArtifacts`